vobject/change_tz.py

```diff
@@ -1,14 +1,14 @@
 """Translate an ics file's events to a different timezone."""
 
 from optparse import OptionParser
-from vobject import icalendar, base
 
-try:
-    import PyICU
-except:
-    PyICU = None
+import pytz
+from dateutil import tz
```

Comment:
What's the minimum dateutil version that has tz?  Is our current 2.4.0 recent enough?

Reply:
| py version| min dateutil needed|
|--------|--------|
| py27, py37, py38, py39| 2.5.0 with warnings|
| py310 and above| 2.7.0|
In all other scenarios, tests fail.